src/modules/playground.ts

```diff
@@ -52,19 +55,28 @@ export class PlaygroundModule extends Module {
 	}
 
 	@listener({ event: 'message' })
-	async onLongPGLink(msg: Message) {
-		const exec = PLAYGROUND_REGEX.exec(msg.content);
+	async onLongPGLink(
+		msg: Message,
+		content = msg.content,
+		noDelete = false,
+		shorten = false,
+	) {
+		const exec = PLAYGROUND_REGEX.exec(content);
 		if (msg.author.bot || !exec || !exec[0]) return;
+		const [originalUrl] = exec;
+		const processedUrl = shorten
+			? await this.shortenPGLink(originalUrl)
+			: originalUrl;
 		const embed = new MessageEmbed()
 			.setColor(TS_BLUE)
-			.setTitle('Shortened Playground link')
+			.setTitle('Shortened Playground Link')
 			.setAuthor(msg.author.tag, msg.author.displayAvatarURL())
-			.setDescription(extractOneLinerFromURL(exec[0]))
-			.setURL(exec[0]);
-		if (exec[0] == msg.content) {
```

Comment:
Was the double on purpose?

Reply:
I doubt it; both are going to be strings